lib/appsignal/hooks/active_support_notifications.rb

```diff
@@ -23,6 +23,21 @@ def self.instrument(name, payload = {})
           end
         end
 
+        instrumenter = ::ActiveSupport::Notifications::Instrumenter
+
+        if instrumenter.method_defined?(:start) && instrumenter.method_defined?(:finish)
+          install_start_finish
+        else
+          install_instrument
```

Comment:
it's not also instrumenting `instrument` because `instrument` also calls `start` and `finish`, so it would double register the event?

Reply:
It only installs instrument when start/end are not present (< Rails 4.x)